IFC 4.3/IfcAlignment_IfcCourse_IfcSectionedSolidHorizontel/IfcAlignment_IfcCourse_IfcSectionedSolidHorizontel.ifc

```diff
@@ -0,0 +1,56 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('Export IFC'), $);
+FILE_NAME ('IFC_Export.ifc', '2021-01-15T07:21:22', (), ('AKG Software Consulting GmbH'), $, $, $);
+FILE_SCHEMA (('IFC4X3'));
+ENDSEC;
+DATA;
+#1 = IFCORGANIZATION('AKG Software Consulting GmbH', $, $, $, $);
+#2 = IFCAPPLICATION(#1, 'Build58', 'VESTRA INFRAVISION', $);
+#3 = IFCOWNERHISTORY($, #2, $, $, $, $, $, 1610691682);
+#4 = IFCDIMENSIONALEXPONENTS(0, 0, 0, 0, 0, 0, 0);
+#5 = IFCUNITASSIGNMENT((#6, #7, #8));
+#6 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#7 = IFCSIUNIT(*, .TIMEUNIT., $, .SECOND.);
+#8 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#9 = IFCAXIS2PLACEMENT3D(#10, $, $);
+#10 = IFCCARTESIANPOINT((0., 0., 0.));
+#11 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-6, #9, $);
+#12 = IFCPROJECTEDCRS('EPSG:25832', 'ETRS89_UTM32', 'EPSG:25832', 'EPSG:25832', 'UTM', '32', #6);
+#13 = IFCMAPCONVERSION(#11, #12, 0., 0., 0., $, $, $);
+#14 = IFCLOCALPLACEMENT($, #15);
+#15 = IFCAXIS2PLACEMENT3D(#16, $, $);
+#16 = IFCCARTESIANPOINT((51.888, -7.639, 0.));
+#17 = IFCPROJECT('1c9L0swYIvW4jEoJxC8FkP', #3, 'Projekt: IfcAlignment_IfcCourse_IfcSectionedSolidHorizontel', 'Projekt: IfcAlignment_IfcCourse_IfcSectionedSolidHorizontel', $, $, $, (#11), #5);
+#18 = IFCSITE('0sSWq56c8qdqaBS9_cIvKc', #3, 'VESTRA', 'VESTRA', $, #14, $, $, $, $, $, $, IFCLABEL('VESTRA'), $);
+#19 = IFCRELAGGREGATES('2Vt1i_2EK8Z4f3gfb1l2V$', #3, $, $, #17, (#18));
+#20 = IFCRELCONTAINEDINSPATIALSTRUCTURE('3jCDKvZ0d7lKBGdztX1NY3', #3, $, $, (#29, #36), #18);
+#21 = IFCPROPERTYSET('3WjALSKistcKhM69FurKtF', $, 'ePSet_MapConversion', $, (#22, #24, #26));
+#22 = IFCPROPERTYSINGLEVALUE('Eastings', $, IFCREAL(0.), $);
+#23 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#24 = IFCPROPERTYSINGLEVALUE('Northings', $, IFCREAL(0.), $);
+#25 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#26 = IFCPROPERTYSINGLEVALUE('OrthogonalHeight', $, IFCREAL(0.), $);
+#27 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#28 = IFCRELDEFINESBYPROPERTIES('1Pax$47IA2japJ5IziiO$Q', #3, $, $, (#17), #21);
+#29 = IFCALIGNMENT('0q2LQAkCNUbK$11RJGlmpP', $, 'Achse Nr. 1', 'Achse Nr. 1', $, $, $, #30, $);
+#30 = IFCALIGNMENTCURVE(#31, #34, $);

```

Comment:
IfcAlignmentCurve is deprecated since IFC4x3_RC2 [Documentation](https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC2/HTML/link/ifcalignmentcurve.htm) 
I guess, the staged sample was modeled against IFC4x3_RC1 ? @pjanck fyi.